src/bit_encoding/decode.rs

```diff
@@ -236,7 +236,7 @@ pub fn decode_expression<I: Iterator<Item = u8>, J: Jet>(
         converted.push(new);
     }
 
-    converted[len - 1].get().map(Arc::clone)
+    converted[len - 1].get().cloned()
```

Comment:
Aside from violating our MSRV, this code is objectively worse because it hides the fact that the `clone` is merely an arc clone.

If clippy is really complaining about this, you probably need to file separate bugs about (a) the lint not respecting MSRV and (b) the lint being stupid.

Reply:
I was struggling to write `.cloned()` in a way that includes `Arc`. The lint is stupid and I disabled it now.